pkg/cluster/embed/autogen_pkger.go

```diff
@@ -22,7 +22,8 @@ func init() {
G9ydGVyJwogICAgLSB0YXJnZXRzOgogICAge3stIHJhbmdlIC5CbGFja2JveEV4cG9ydGVyQWRkcnN9fQogICAgICAtICd7ey59fScKICAgIHt7LSBlbmR9fQogICAgICBsYWJlbHM6CiAgICAgICAgZ3JvdXA6ICdibGFja2JveF9leHBvcnRlcicKICAgIHJlbGFiZWxfY29uZmlnczoKICAgICAgLSBzb3VyY2VfbGFiZWxzOiBbX19hZGRyZXNzX19dCiAgICAgICAgdGFyZ2V0X2xhYmVsOiBfX3BhcmFtX3RhcmdldAogICAgICAtIHNvdXJjZV9sYWJlbHM6IFtfX3BhcmFtX3RhcmdldF0KICAgICAgICB0YXJnZXRfbGFiZWw6IGluc3RhbmNlCiAgICAgIC0gdGFyZ2V0X2xhYmVsOiBfX2FkZHJlc3NfXwogICAgICAgIHJlcGxhY2VtZW50OiB7ey5CbGFja2JveEFkZHJ9fQp7ey0gcmFuZ2UgJGFkZHIgOj0gLkJsYWNrYm94RXhwb3J0ZXJBZGRyc319CiAgLSBqb2JfbmFtZTogImJsYWNrYm94X2V4cG9ydGVyX3t7JGFkZHJ9fV9pY21wIgogICAgc2NyYXBlX2ludGVydmFsOiA2cwogICAgbWV0cmljc19wYXRoOiAvcHJvYmUKICAgIHBhcmFtczoKICAgICAgbW9kdWxlOiBbaWNtcF0KICAgIHN0YXRpY19jb25maWdzOgogICAgLSB0YXJnZXRzOgogICAge3stIHJhbmdlICQuTW9uaXRvcmVkU2VydmVyc319CiAgICAgIC0gJ3t7Ln19JwogICAge3stIGVuZH19CiAgICByZWxhYmVsX2NvbmZpZ3M6CiAgICAgIC0gc291cmNlX2xhYmVsczogW19fYWRkcmVzc19fXQogICAgICAgIHJlZ2V4OiAoLiopKDo4MCk/CiAgICAgICAgdGFyZ2V0X2xhYmVsOiBfX3BhcmFtX3RhcmdldAogICAgICAgIHJlcGxhY2VtZW50OiAkezF9CiAgICAgIC0gc291cmNlX2xhYmVsczogW19fcGFyYW1fdGFyZ2V0XQogICAgICAgIHJlZ2V4OiAoLiopCiAgICAgICAgdGFyZ2V0X2xhYmVsOiBwaW5nCiAgICAgICAgcmVwbGFjZW1lbnQ6ICR7MX0KICAgICAgLSBzb3VyY2VfbGFiZWxzOiBbXQogICAgICAgIHJlZ2V4OiAuKgogICAgICAgIHRhcmdldF9sYWJlbDogX19hZGRyZXNzX18KICAgICAgICByZXBsYWNlbWVudDoge3skYWRkcn19Cnt7LSBlbmR9fQoKe3stIGlmIC5ETU1hc3RlckFkZHJzfX0KICAtIGpvYl9uYW1lOiAiZG1fbWFzdGVyIgogICAgaG9ub3JfbGFiZWxzOiB0cnVlICMgZG9uJ3Qgb3ZlcndyaXRlIGpvYiAmIGluc3RhbmNlIGxhYmVscwogICAgc3RhdGljX2NvbmZpZ3M6CiAgICAtIHRhcmdldHM6CiAgICB7ey0gcmFuZ2UgLkRNTWFzdGVyQWRkcnN9fQogICAgICAtICd7ey59fScKICAgIHt7LSBlbmR9fQp7ey0gZW5kfX0KCnt7LSBpZiAuRE1Xb3JrZXJBZGRyc319CiAgLSBqb2JfbmFtZTogImRtX3dvcmtlciIKICAgIGhvbm9yX2xhYmVsczogdHJ1ZSAjIGRvbid0IG92ZXJ3cml0ZSBqb2IgJiBpbnN0YW5jZSBsYWJlbHMKICAgIHN0YXRpY19jb25maWdzOgogICAgLSB0YXJnZXRzOgogICAge3stIHJhbmdlIC5ETVdvcmtlckFkZHJzfX0KICAgICAgLSAne3sufX0nCiAgICB7ey0gZW5kfX0Ke3stIGVuZH19Cgp7ey0gaWYgLlJlbW90ZVdyaXRlVVJMfX0KcmVtb3RlX3dyaXRlOgogICAtIHVybDoge3suUmVtb3RlV3JpdGVVUkx9fQogICAgIHF1ZXVlX2NvbmZpZzoKICAgICAgIHt7LSBpZiAuQ2FwYWNpdHl9fQogICAgICAgY2FwYWNpdHk6IHt7LkNhcGFjaXR5fX0KICAgICAgIHt7LSBlbmR9fQogICAgICAge3stIGlmIC5NYXhTaGFyZHN9fQogICAgICAgbWF4X3NoYXJkczoge3suTWF4U2hhcmRzfX0KICAgICAgIHt7LSBlbmR9fQogICAgICAge3stIGlmIC5NaW5TaGFyZHN9fQogICAgICAgbWluX3NoYXJkczoge3suTWluU2hhcmRzfX0KICAgICAgIHt7LSBlbmR9fQogICAgICAge3stIGlmIC5NYXhTYW1wbGVzUGVyU2VuZH19CiAgICAgICBtYXhfc2FtcGxlc19wZXJfc2VuZDoge3suTWF4U2FtcGxlc1BlclNlbmR9fQogICAgICAge3stIGVuZH19CiAgICAgICB7ey0gaWYgLkJhdGNoU2VuZERlYWRsaW5lfX0KICAgICAgIGJhdGNoX3NlbmRfZGVhZGxpbmU6IHt7LkJhdGNoU2VuZERlYWRsaW5lfX0KICAgICAgIHt7LSBlbmR9fQogICAgICAge3stIGlmIC5NaW5CYWNrb2ZmfX0KICAgICAgIG1pbl9iYWNrb2ZmOiB7ey5NaW5CYWNrb2ZmfX0KICAgICAgIHt7LSBlbmR9fQogICAgICAge3stIGlmIC5NYXhCYWNrb2ZmfX0KICAgICAgIG1heF9iYWNrb2ZmOiB7ey5NYXhCYWNrb2ZmfX0KICAgICAgIHt7LSBlbmR9fQp7ey0gZW5kfX0KCnt7LSBpZiAuUmVtb3RlUmVhZFVSTH19CnJlbW90ZV9yZWFkOgogICAtIHVybDoge3suUmVtb3RlUmVhZFVSTH19Cnt7LSBlbmR9fQo="
```

Comment:
Just let make pkger regenerate this file, don't try to edit it.

Reply:
fixed